packages/expo-cli/commands/install.js

```diff
@@ -63,6 +65,30 @@ const checkManifest = async (dir) => {
   }
 }
 
+const selectVersion = async (dir) => {
+  try {
+    const pkg = JSON.parse(await promisify(readFile)(join(dir, 'package.json'), 'utf8'))
+    const expoVersion = pkg.dependencies.expo
+    const isPre33 = (expoVersion && !semver.gte(semver.minVersion(expoVersion), '33.0.0'))
+    const { version } = await prompts({
+      type: 'text',
+      name: 'version',
+      message: isPre33
+        ? 'It looks like you’re using a version of Expo SDK <33. The latest version of Bugsnag works with SDK >= 33 so it’s recommended that you install the last version of Bugsnag that supported your Expo version: v6.3.0'
+        : 'If you want the latest version of @bugsnag/expo hit enter, otherwise type the version you want',
+      initial: isPre33 ? '6.3.0' : 'latest',
+      validate: str => {
+        if (semver.valid(str)) return true
+        if (semver.validRange(str)) return true
+        return 'Version must be: a valid semver version/range or "latest"'
```

Comment:
This doesn't seem to support `latest` as an input - fails with:
```
Version must be: a valid semver version/range or "latest"
```

Reply:
ah yes good catch, need to check `str === 'latest'`  :+1: